{{ cookiecutter.dir_name }}/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
+    }
+}
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
```

Comment:
Erm, that's odd.
I hate to say "worked in my machine", but I did tried it myself earlier before commenting and it downloaded for me (I tried with `25.0.1` which I didn't have, and it took a long while but it installed it).

In fact I've just tried again, by deleting `$ANDROID_HOME/build-tools/25.0.2` and it seems to be working:

```
Preparing "Install Android SDK Build-Tools 25.0.2".
> Configuring > 1/1 projects
```

Taking a long time in my current slow connection, though.
Could you try again sometime? (maybe instead of deleting the directory, just trying some other version you don't have installed)